src/s3.rs

```diff
@@ -11278,10 +11289,6 @@ impl<P> S3Client<P> where P: ProvideAwsCredentials {
             }
             _ => {
                 println!("Error: Status code was {}", status);
-                let mut body = String::new();
```

Comment:
This debug info is now thrown out - I don't think that's what we want.  I need to get the result body and log that out to keep our debugging stuff in.  Happens in a few places.